### ISSUE TITLE
not good in test

### ISSUE THREAD

**Issue by ReySadeghi** [UKPLab/sentence-transformers#1137]
Hi,
I'm trying to fine-tune a Persian BERT model on a binary classification dataset prepared on Persian.(paraphrase, non-paraphrase)
the thing is happening is before first epoch is completely done, I got the best avg-precision and the model is saving with 80% accuracy. but in test phase it's functionality is very bad and similarity scores are wrong among 2 sentences. I mean the sentences that are not in similar with my query, have the most score!
do you have any idea?

**Comment by nreimers** [comment]
Some more information is needed on your setup

**Comment by ReySadeghi** [comment]
I ran it on ubuntu, 
python3.7 
sentence-transformers==0.4.1
torch:1.6.0+cu101

my pretrained model: https://huggingface.co/HooshvareLab/bert-base-parsbert-ner-uncased
